src/components/BackLink.jsx

```diff
@@ -0,0 +1,32 @@
+import { Link } from 'react-router-dom';
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import dictIcons from '../img/icons/index';
+import IconComponent from '../components/Icon';
+
+const BACK_CAPTION = 'Back to data overview';
+
+export const BackComponent = styled.div`
+  display: inline;
+  padding-right: 15px;
+`;
+
+class BackLink extends Component {
+  render() {
+    return (
+      <Link to={'/files'}>
```

Comment:
this BackLink component:
- if you want to keep it in `src/components` folder, it should have a parameter `backlink`
 with a link to page where you want to go back.
- otherwise, it should locate in a page of `src/CoreMetadata` (where you use it with hardcoded value `/files`